### ISSUE TITLE
Change startingPoint default props on PieChart (donut)

### ISSUE THREAD

**Issue by NemyaNation** [Abhinandan-Kushwaha/react-native-gifted-charts#132]
Just some  ideas for you @Abhinandan-Kushwaha 

It would be smarter to start your donut/pieChart from the center top (0 degrees) instead of the current (90 degrees)

The reason for this is because usually in most donut charts, they are displayed in descending order, meaning that the right hand side of the entire donut graph, will most of the time only show one label (< 90 degress)

And since most people output their labels LTR (Left to right is the standard), this would mean that the most clusterd position usually will be the right hand side of the donut chart.

This can be avoided in most cases however if you start at (0 degrees)

Look at this image to further understand my point:

![Screenshot_2022-04-10-01-06-01-708_com tcgfintech tcgprice](https://user-images.githubusercontent.com/17049683/162595793-47a5f639-c6fd-4046-939a-c7115ab362fb.jpg)

---

As you can see my data is in descending order, light blue is biggest (starting at 90*), then dark blue then purple. Unfortunately this means that the smaller labels are congested since they are all rightToLeft and therefore cut off from the screen entirely.

This could be avoided if the're final posiiton was somewhere near the top (the case if you start at 0degrees)

---

Usually for (LTR) we want as least labels as possible on the right (avoid overflow issues like we see above).

Let me know if this at all makes any sense to you, and if not I can expand my point. (should be a simple edit that I think is better as the default (I can't see any use-cases for starting at 90degrees instead)

Also, zIndex of the labels should always be higher than the *slices*, I can't see a usecase where you would want to hide the label if it overlaps onto another (because it's starting point is always visible)

**Comment by Abhinandan-Kushwaha** [comment]
Hi @NemyaNation 👋
Thanks for requesting this feature. It is now available in version `1.0.18`🎉

We can now set the starting angle using the `initialAngle` prop.
It takes angle value in radians. The default value is 0. The angles are distributed differently from the usual coordinate system. It is illustrated below-

<img width="435" alt="pie" src="https://user-images.githubusercontent.com/20596944/163213821-6ab507b8-dcc2-4bf2-a0e3-952f6190ecfd.png">

**Comment by NemyaNation** [comment]
Hey @Abhinandan-Kushwaha thanks for the quick response!

Can we get a quick example on how this works?

I'm not quire sure what the input for this prop should be?

I've tried 

```
initialAngle={'0.785398rad'}
```

**Comment by NemyaNation** [comment]
Also @Abhinandan-Kushwaha 

```
Also, zIndex of the labels should always be higher than the slices, I can't see a usecase where you would want to hide the label if it overlaps onto another (because it's starting point is always visible)
```

**Comment by Abhinandan-Kushwaha** [comment]
Hi @NemyaNation 👋
The `initialAngle` should be passed in radians as a number not as a string. For example-
```js
initialAngle = {Math.PI / 2}
```

Also, the zIndex issue with labels is now solved in version `1.0.19`🎉

**Comment by NemyaNation** [comment]
Brilliant! zIndex for pieChart labels is now fixed and I've gotten the initalAngle to perform!

**Comment by Abhinandan-Kushwaha** [comment]
Hi @NemyaNation 👋
To make the library more performant, we have **removed** `canvas` and `webview` from the library. This is a great enhancement to the library implemented in version `1.2.0`🎉

Earlier we were rendering Pie and Donut charts using canvas. But now, even they rendered using **svg** (just like Bar and Line charts)

Please consider using the latest version of the library. Also, please remove `react-native-canvas` and `react-native-webview` from your project, if you had installed them for charts.

**Comment by NemyaNation** [comment]
Brilliant 🎉🎉🎉 Thanks for continuing to make this library even greater! :)
